lib/consoletest.pm

```diff
@@ -11,6 +11,11 @@ use known_bugs;
 use version_utils qw(is_public_cloud is_openstack);
 use utils;
 
+my %avc_record = (
+    start => 0,
```

Comment:
Is this shared between all instances of `consoletest`? If not, all tests will start at 0

Reply:
Ok - perl is still an alien language to me